scripts/zones/Apollyon/IDs.lua

```diff
@@ -18,16 +18,22 @@ zones[xi.zone.APOLLYON] =
         LOGIN_CAMPAIGN_UNDERWAY       = 7002, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!
         LOGIN_NUMBER                  = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
         MEMBERS_LEVELS_ARE_RESTRICTED = 7023, -- Your party is unable to participate because certain members' levels are restricted.
-        TIME_EXCEEDED                 = 7065, -- You have exceeded the time limit. Exiting Limbus now.
```

Comment:
Was this unused?

Reply:
This is replaced with `ID.text.TIME_IN_THE_BATTLEFIELD_IS_UP` which is shared between all battlefield zones.